ci/jenkins_build

```diff
@@ -1,14 +1,31 @@
 #!/bin/bash
 set -euo pipefail
 
+WAIT_TIMEOUT="${WAIT_TIMEOUT:-600}"
+
 TOOLS_IMAGE_NAME="conjur-gke-marketplace-kubectl"
 TMP_DIR="$(pwd)/.tmp"
 BIN_DIR="${TMP_DIR}/bin"
 
+function announce {
+    echo "==================================================="
+    echo "$1"
+    echo "==================================================="
+}
+
+function final_result {
+    echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
+    echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
+    echo "$1"
+    echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
+    echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
+}
```

Comment:
This is probably unnecessary vs the standard announce (even the announce is probably overkill)